src/brain-map.ts

```diff
@@ -0,0 +1,30 @@
+import { GameState } from './game-state';
+
+export class BrainMap {
+  state: GameState[] = new Array(20000);
+
+  constructor(initState: GameState[] = []) {
+    initState.forEach((move, i) => {
```

Comment:
What is this doing exactly and how long does it take?

Reply:
this won't take long and it only happens once per training run

this is setting the initial state from the stored brain and setting it to the state array